### DISCUSSION TITLE
Cluster relocation

### DISCUSSION THREAD

**Discussion by sentaycan** [backube/volsync#1266] [Q&A]
Hello, I am copying my PVC to a cluster in a different region as below, but now I want to delete my source cluster and use my destination cluster as source cluster, what is the most optimal way to do this?

`kubectl --context west2 -n mysql create -f - <<EOF
apiVersion: volsync.backube/v1alpha1
kind: ReplicationDestination
metadata:
  name: mysql
spec:
  rsyncTLS:
    copyMethod: Snapshot
    capacity: 8Gi
    accessModes: ["ReadWriteOnce"]
    storageClassName: gp3-csi
    volumeSnapshotClassName: csi-aws-vsc
    keySecret: east-west
    serviceType: LoadBalancer
EOF`
`kubectl --context east2 -n mysql create -f - <<EOF
apiVersion: volsync.backube/v1alpha1
kind: ReplicationSource
metadata:
  name: mysql
spec:
  sourcePVC: data-mysql-0
  trigger:
    schedule: "*/5 * * * *"
  rsyncTLS:
    keySecret: east-west
    address: abad73aa2ca4441ed8c9e13f1095c453-95258c1d3ff95327.elb.us-west-2.amazonaws.com
    copyMethod: Snapshot
    volumeSnapshotClassName: csi-aws-vsc
EOF`

**Comment by tesshuflower** [comment]
@sentaycan
Once you have a PVC on the destination and you're happy with it (i.e. you've created a PVC from the snapshot created by the replicationdestination and it contains your data), then you will just have to do the reverse to setup a replication going in the other way.

That is, you'll need to create a replicationdestination on the old source (new destination) cluster, and your new PVC on the destination (that you now want to be the source) should be the sourcePVC for a new replicationsource.

**Comment by tesshuflower** [comment]
One note - on the new destination cluster, since it contains the old Source PVC, you can use this as your destinationPVC in your replicationdestination if you wish - this means the initial sync back from new source to new destination should be quicker as the destination PVC already contains a lot of the data.

**Comment by sentaycan** [comment]
Thank you, but I want to connect the PVC in DestinationCluster to my application, how can I do this, I couldn't do it :(

**Comment by sentaycan** [comment]
And my question was actually this.
I want to delete my source cluster and use the destination cluster as the main cluster.
What is the most optimal way to do this?

**Comment by tesshuflower** [comment]
@sentaycan I've converted this to a discussion as I don't think there's a specific code issue we're talking about here, just recommendations on usage.

When it comes to using your PVC at the destination side with your application, you will need to first create a PVC from the volumesnapshot that was created by your replicationdestination.  

Have you done this step already?  If so,  you will just need to make sure you startup your app and that it uses the PVC (for example if it's a deployment that mounts the PVC, make sure the pvc is named correctly and in the correct namespace).

**Reply by sentaycan** [reply]
So when I want to use my destination cluster as main cluster, all I have to do is connect the PVC in the Destination Cluster to my application and stand it up.This is the optimum way.

My second question is that I am looking for volsync default technical values but I could not find anything in the document.

Thank you very much for your help.

**Reply by sentaycan** [reply]
Sir ?
